include/industry_standard/spdm.h

```diff
@@ -533,6 +533,7 @@ typedef struct {
 /* SPDM GET_MEASUREMENTS request Attributes */
 #define SPDM_GET_MEASUREMENTS_REQUEST_ATTRIBUTES_GENERATE_SIGNATURE 0x00000001
 #define SPDM_GET_MEASUREMENTS_REQUEST_ATTRIBUTES_RAW_BIT_STREAM_REQUESTED 0x00000002
+#define SPDM_GET_MEASUREMENTS_REQUEST_ATTRIBUTES_NEW_MEASUREMENT_REQUESTED 0x00000004
```

Comment:
why change this?

Reply:
I did not make this change, it was part of the commit https://github.com/DMTF/libspdm/commit/052bc7b1f02e3d0538d07525359b9e29e4f7738a

Reply:
![image](https://github.com/DMTF/libspdm/assets/12147155/eb9b42f9-1943-4c0b-8354-db08e0b9d6f4)

That is weird. Look at this!

If you dont need it, then please remove it.

Reply:
I must have done the rebase wrong. I will do it again to remove the extra commits.